client/extensions/woocommerce/app/settings/payments/stripe/payment-method-stripe-setup-dialog.js

```diff
@@ -83,9 +135,45 @@ class PaymentMethodStripeSetupDialog extends Component {
 					onSelectCreate={ this.onSelectCreate }
 					onSelectConnect={ this.onSelectConnect }
 				/>
+				{ this.possiblyRenderNotice() }
+				<QuerySettingsGeneral siteId={ siteId } />
 			</Dialog>
 		);
-	}
+	};
+}
+
+function mapStateToProps( state ) {
+	const email = getCurrentUserEmail( state );
+	const site = getSelectedSiteWithFallback( state );
+	const siteId = site.ID || false;
+
+	const error = getError( state, siteId );
+	const isCreating = getIsCreating( state, siteId );
+
+	const isLoadingAddress = areSettingsGeneralLoading( state, site.ID );
```

Comment:
Minor: Pass `siteId` here and in `getStoreLocation`, to be consistent with your other selector calls here.

Reply:
Merge magic (or lack of it). Fixed in https://github.com/Automattic/wp-calypso/pull/19292/commits/4f0f8f8f26f28b59184594aaf729dbca2265f243